api/app/src/routes/medical/organization.ts

```diff
@@ -32,6 +35,15 @@ router.post(
   "/",
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
+
+    // Initialize the account before creating the organization
+    await accountInit(cxId);
+
+    // Allow MAPI access in sandbox
+    if (Config.isSandbox()) {
+      await allowMapiAccess(cxId);
```

Comment:
Users need MAPI access before being able to create an Org, `/medical/v1/*` has `checkMapiAccess`.

Instead of adding it here, maybe we just add a logic to always allow MAPI access on `checkMapiAccess` when sandbox, then we don't need this here? Just thinking how to minimize spreading the logic about MAPI access and sandbox across the codebase.

Reply:
Smart thats a good point. Ill update that